### ISSUE TITLE
Artificial Neural Networks - The Complete Noob Guide.

### ISSUE THREAD

**Issue by DakshMiglani** [pydatadelhi/talks#49]
- Abstract (2-3 lines)
A Complete guide for the absolute beginner or Noob for teaching them the fundamentals of Artificial Neural Networks and the Process of How the Function In General.

- Brief Description and Contents to be covered
Will be covering the basics of artificial neural networks such as weights, biases, inputs, labels, features, activation function, loss function.

- Pre-requisites for the talk
None

- Time required for the talk
15 Minutes

- Link to slides
https://docs.google.com/presentation/d/1NMrrmDqY6q3ukr10gYazOOrQp6d8Gd6EXGfNG6AJAbg/edit?usp=sharing

- Will you be doing hands-on demo as well?
I would but in another talk probably next time :)

- Link to ipython notebook (if any)

- About yourself
I'm a 15 Year Old Wizard, Who uses Python-JavaScript and C++ for Magic. I'm also a Deep-Learning Student and a Space Enthusaist and I'm programming for 5 years now.

- Are you comfortable if the talk is recorded and uploaded to PyData Delhi's YouTube channel ?
Sure :)

- Any query ?
No.

**Comment by MSanKeys963** [comment]
Cc : @manojpandey

**Comment by MSanKeys963** [comment]
@DakshMiglani Can you present this on next Meetup?

**Comment by DakshMiglani** [comment]
@MSanKeys963 On The Upcoming Sunday right? (6th of August)

**Comment by MSanKeys963** [comment]
Yeah.

**Comment by DakshMiglani** [comment]
@MSanKeys963 Would love to but just have to setup some alarms to wake me up that early, Don't worry I'll be there 👍

**Comment by DakshMiglani** [comment]
@MSanKeys963 Let me know the schedule.

**Comment by manojpandey** [comment]
Hi @DakshMiglani - the slides are too much filled with text. I'll suggest you to keep the minimum possible, and remove everything to your speaker notes. Please update the slides ASAP, if possible:)

Otherwise we can take this up as a lightning talk )

**Comment by DakshMiglani** [comment]
@manojpandey Yea, I'll do that and update you by 6pm IST with slides.

**Comment by DakshMiglani** [comment]
Here is the file, I'll be writing the speaker notes tommorow and they would arrive around this time:
https://docs.google.com/presentation/d/1nmhzEMbzhl3IzOHQ4ukqQjYiYCF_oideFo2jkxwPDhI/edit?usp=sharing
